packages/blockchain-extension/extension/commands/addGatewayCommand.ts

```diff
@@ -48,12 +49,12 @@ export async function addGateway(): Promise<{} | void> {
 
         if (gatewayMethod === UserInputUtil.ADD_GATEWAY_FROM_ENVIRONMENT) {
 
-            const environments: FabricEnvironmentRegistryEntry[] = await FabricEnvironmentRegistry.instance().getAll(false, false, true);
+            const environments: FabricEnvironmentRegistryEntry[] = await FabricEnvironmentRegistry.instance().getAll([], [EnvironmentFlags.ANSIBLE]);
             if (environments.length === 0) {
                 throw new Error(`No environments to choose from. Gateways cannot be created from managed Ansible or local environments.`);
```

Comment:
`No environments to choose from. Gateways cannot be created from Ansible or local environments.`